tools/VscPreviewExtension/build.ps1

```diff
@@ -0,0 +1,40 @@
+param(
+    [string] $command = "package",
+    [string] $version
+)
+
+################################################################################################
+# Usage:
+# Run build.ps1
+#   [-command Command]: package or publish, default to be package
+#   [-version Version]: Publish version which should be newer than published version
+################################################################################################
+$ErrorActionPreference = 'Stop'
+$scriptPath = $MyInvocation.MyCommand.Path
+$scriptHome = Split-Path $scriptPath
+$configuration = "Release"
+$framework = "net46"
+$HttpServiceFolder = "DfmHttpService"
+$extensionFolder = "VscPreviewExtension"
```

Comment:
known issue, resolved.

---
In reply to: [109098566](https://github.com/dotnet/docfx/pull/1511#discussion_r109098566) [](ancestors = 109098566)